manim/animation/transform.py

```diff
@@ -15,17 +17,15 @@
 from ..utils.rate_functions import squish_rate_func
 
 
+@attr.s(auto_attribs=True, eq=False)
 class Transform(Animation):
-    CONFIG = {
-        "path_arc": 0,
-        "path_arc_axis": OUT,
-        "path_func": None,
-        "replace_mobject_with_target_in_scene": False,
-    }
+    path_arc: float = 0
+    path_arc_axis: np.ndarray = OUT
+    path_func: tp.Any = None
```

Comment:
it can't be any as it expects a function, so at least a Callable. We gotta work on inferring the expected function signature though  (TODO)
```suggestion
    path_func: tp.Callable = None
```